e2etest/clean/clean_distributor.go

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package main
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm"
+	"log"
+	"sort"
+	"strconv"
+)
+
+func main() {
+	//set up aws go sdk config
+	awsConfig := aws.Config{}
+	testSession, err := session.NewSession(&awsConfig)
+	if err != nil {
+		log.Fatalf("Error creating session %v", err)
+	}
+	// set up aws go sdk ssm client
+	ssmClient := ssm.New(testSession, &awsConfig)
```

Comment:
Same for this client. https://docs.aws.amazon.com/sdk-for-go/api/service/ssm/#New

```
// Create a SSM client from just a session.
svc := ssm.New(mySession)
```